concrete/authentication/facebook/hooked.php

```diff
@@ -0,0 +1,24 @@
+<?php defined('C5_EXECUTE') or die('Access Denied.'); ?>
```

Comment:
What is this file?

Reply:
It's the same as https://github.com/concrete5/concrete5/blob/develop/concrete/authentication/community/hooked.php
It's rendered when someone attach his account. https://github.com/concrete5/concrete5/blob/develop/concrete/single_pages/account/edit_profile.php

It's the `Detach your facebook account` section on my image.
![untitled](https://user-images.githubusercontent.com/2462951/31259285-96dfd496-aa80-11e7-848f-ee6cdf6eb862.png)

Reply:
:+1: thanks for the refresher on that